logproof/src/fields.rs

```diff
@@ -48,10 +48,15 @@ where
  *
  * SEAL uses Q =
  * 0x7fffffd8001 * 0x7fffffc8001 * 0xfffffffc001 * 0xffffff6c001 * 0xfffffebc001
+ *
+ * This can be derived by running
+ * `CoefficientModulus::bfv_default(8192, SecurityLevel::TC128)`
+ * or by running the underlying SEAL function
+ * `CoeffModulus::BFVDefault(8192, sec_level_type::tc128)`
  */
 #[derive(MontConfig)]
-#[modulus = "421249101157150430150591791601812858371395928330411389778873040897"]
 #[generator = "3"]
+#[modulus = "23945240908173643396739775218143152511335532357255169"]
```

Comment:
The higher two moduli (4096 and 8192) had multiple components (say N), of which the encryption is done over the product of only the first N - 1 moduli. The final special modulus is only used for modulus switching, which reduces initial noise on encryption in some schemes. 

See https://crypto.stackexchange.com/a/87971